src/http-connection.md

```diff
@@ -54,6 +55,18 @@ Note that when using extension keys, you need to replace any dots with `&46;` be
 #### Sorting With Improved Performance
 You may find that changing the sort parameter can vary the time it takes a query to run, especially when you have a large number of models. You can take advantage of database indexes to improve performance, more information is available about [using indexes via Mongo's documentation](https://docs.mongodb.com/manual/indexes/). If utilising indexes doesn't have the required performance improvement, you can instead utilise [BI tools](../guides-retrieving).
 
+### Search Parameter
```

Comment:
James will want an example request here so where it currently says "So if the value for this parameter is `exampleSearchString`, it will be transformed into filter as shown below." I think we should perhaps reword that to say "So if the value for this parameter is `exampleSearchString`, as shown in the request below, it will be transformed into filter as shown below."